modules/install-acs-operator.adoc

```diff
@@ -33,7 +33,7 @@ If you choose manual updates, when a newer version of the Operator is available,
 +
 [IMPORTANT]
 ====
-If you choose manual updates, you must update the {product-title-short} Operator in all secured clusters when you update the {product-title-short} Operator in the cluster where Central is installed. The secured clusters and the cluster where Central is installed must have the same version to ensure optimal functionality.
+If you choose manual updates, you must update the {product-title-short} Operator in all secured clusters when you update the {product-title-short} Operator in the cluster where Central is installed. The secured clusters and the cluster where Central is installed must have the same version to ensure optimal functionality. For {product-title-managed-short}, use the latest version of the software that is available. (The software version for Central is automatically updated.)
 ====
```

Comment:
> This gets rendered on the page dedicated to the installation for self-managed. [63530--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/install-central-ocp#install-acs-operator_install-central-ocp](https://63530--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/install-central-ocp#install-acs-operator_install-central-ocp)

I know - I was trying to avoid using ifdef statements because it can get confusing, but I will try to see if I can add them to get it not to show on self-managed.